frontend/translations/es.po

```diff
@@ -190,10 +197,18 @@ msgstr ""
 "Al crear una nueva cuenta, aceptas nuestros [términos de servicio](%s) y "
 "[política de privacidad](%s)."
 
+msgid "auth.terms-and-privacy-agreement"
+msgstr ""
+"Yo acepto los [términos de servicio](%s) y la [política de privacidad](%s)."
```

Comment:
This is markdown format, the translation in penpot don't understand that so the rendered result is:
![pwserver_or_not](https://github.com/penpot/penpot/assets/1579633/71a14c0d-6bb3-4e2b-9ae9-f239f5a7b371)

Probably it should be something like `Yo acepto los <a href = "%s">términos de servicio....`